sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -364,6 +377,33 @@ private async Task<Response<DetectLanguageResultCollection>> DetectLanguageBatch
             }
         }
 
+        // Overload Method for New Swagger
+        private async Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(LanguageDetectionAnalysisInput batchInput, TextAnalyticsRequestOptions options, CancellationToken cancellationToken)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TextAnalyticsClient)}.{nameof(DetectLanguageBatch)}");
+            scope.Start();
+
+            try
+            {
+                var analyzeLanguageDetection = new AnalyzeTextLanguageDetectionInput { AnalysisInput = batchInput };
+                analyzeLanguageDetection.Parameters.ModelVersion = options.ModelVersion;
+                analyzeLanguageDetection.Parameters.LoggingOptOut = options.DisableServiceLogs;
```

Comment:
curious why the `Parameters` are treated differently than `AnalysisInput`

Reply:
I updated the Parameter construction to match the AnalysisInput